spec/support/shared/shopify_request.rb

```diff
@@ -14,18 +14,17 @@
     ShopifyAPI::Base.user = nil
   end
 
-  def mock_request(endpoint, extension)
+  def mock_request(file, endpoint, extension)
```

Comment:
How would you go about the case we're facing where the endpoint is `orders/{order_id}/transactions`